src/ts/components/playbacktogglebutton.ts

```diff
@@ -73,6 +89,7 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
     player.on(player.exports.PlayerEvent.Warning, (event: WarningEvent) => {
       if (event.code === player.exports.WarningCode.PLAYBACK_COULD_NOT_BE_STARTED) {
         this.isPlayInitiated = false;
+        firstPlay = true;
```

Comment:
why do we need to re-set here? isn't the player in fullscreen already?

Reply:
This warning should only happen with autoplay. By having this here, we enable the enter fullscreen when the user _actually_, manually starts the initial playback (because autoplay was blocked).

If I remove this, the `firstPlay` would be set to `false` in the `Play` event which is also triggered with autoplay.

I see that both options are somewhat reasonable. Keep in mind that in order to remove this also from the `hugeplaybacktogglebutton.ts`, we'd need to introduce yet another flag because the `firstPlay` there was in place before and is also used for other stuff, so we can't just remove that there.

Reply:
Fine as is.